modules/EvseV2G/v2g_server.cpp

```diff
@@ -380,10 +377,11 @@ int v2g_handle_connection(struct v2g_connection* conn) {
 
     do {
         /* setup for receive */
-        conn->stream.buffer = 0;
-        conn->stream.capacity = 0; // Set to 8 for send and 0 for recv
-        conn->buffer_pos = 0;
+        conn->stream.data[0] = 0;
+        conn->stream.bit_count = 0;
+        conn->stream.byte_pos = 0;
```

Comment:
Might use `exi_bitstream_reset()`, but okay if it works.

Reply:
Good point :+1: But I think we can delete line 381 and 382 because in line 383 `exi_bitstream_init()` is called. In the init function `bit_count` and `byte_pos` is set to 0 anyway.

Reply:
Done